src/main/java/com/stablekernel/standardlib/RxFragment.java

```diff
@@ -28,6 +28,12 @@ protected void showBlockingProgress() {
         showBlockingProgress(null);
     }
```

Comment:
If you want it to not be cancelable by default, then you'll need this default method to call `showBlockingProgress(false);`